src/handlers/shared/check-unassigns.ts

```diff
@@ -0,0 +1,20 @@
+import { Context } from "../../types";
+
+export async function wasPreviouslyUnassigned(
+  context: Context,
+  senderLogin: Context["payload"]["sender"],
+  issueNumber: Context["payload"]["issue"]
```

Comment:
naming convention for these args is off slightly so it's being used in error in the return statement below I think